Cargo.toml

```
@@ -45,4 +45,4 @@ governor = "0.6.3"
 tokio-util = "0.7.10"
 chrono = "0.4"
 chrono-humanize = "0.2.3"
-dashmap = "5.5.3"
+dashmap = "5.5.3"
```

Minor suggestion, but the newline got dropped in your commit
```suggestion
dashmap = "5.5.3"

```